spec/policies/presentation_policy_spec.rb

```diff
@@ -24,16 +26,56 @@
       end
     end
 
+    context 'the person is an admin' do
+      subject { described_class.new(admin, presentation) }
+      let(:admin) { FactoryGirl.build(:person, :admin) }
+      let(:presentation) { FactoryGirl.build(:presentation) }
+
+      describe '.update?' do
+        it 'returns true' do
+          expect(subject.update?).to eq(true)
+        end
+      end
+
+      describe '.edit?' do
+        it 'calls the .update? method' do
+          expect(subject).to receive(:update?)
+          subject.edit?
+        end
+      end
+    end
+
     context 'the person is a non admin' do
-      subject { PresentationPolicy.new(non_admin, 'any_object') }
+      subject { PresentationPolicy.new(non_admin, presentation) }
       let(:non_admin) { FactoryGirl.build(:person) }
+      let(:presentation) { FactoryGirl.build(:presentation) }
+
+      describe '.update?' do
+        context %(the presentation being taken action on
```

Comment:
Yeah rubocop likes double like strings as a percent literal. I'll update the rules though.